backend/typescript/services/implementations/schedulingService.ts

```diff
@@ -321,6 +321,149 @@ class SchedulingService implements ISchedulingService {
     }
   }
 
+  async sendAdminEmailVerificationAfterDonationScheduled(
+    donor: UserDonorDTO,
+    schedule: SchedulingDTO,
+  ): Promise<void> {
+    if (!this.emailService) {
+      const errorMessage =
+        "Attempted to call sendAdminEmailVerificationAfterDonationScheduled but this instance of SchedulingService does not have an EmailService instance";
+      Logger.error(errorMessage);
+      throw new Error(errorMessage);
+    }
+
+    try {
+      const { firstName, lastName } = donor;
+      // HARD CODED TEST ADMIN EMAIL
+      const adminEmail = "jessiepeng@uwblueprint.org";
```

Comment:
can you create a ticket for us to remember to change it back? 

I would also lift this to be a static constant for the whole class, something like:
`static TEMP_ADMIN_EMAIL = "jessiepeng@uwblueprint.org"`